lib/suspenders/app_builder.rb

```diff
@@ -245,6 +259,10 @@ def configure_unicorn
       copy_file 'unicorn.rb', 'config/unicorn.rb'
     end
 
+    def configure_unicorn_mongoid
+      copy_file 'unicorn_mongoid.rb', 'config/unicorn.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.